images/iqsharp-base/Dockerfile

```diff
@@ -93,7 +93,7 @@ ENV PATH=$PATH:${HOME}/dotnet:${HOME}/.dotnet/tools \
 # Install IQ# and the project templates, using the NuGet packages from the
 # build context.
 ARG IQSHARP_VERSION
-RUN dotnet new -i "Microsoft.Quantum.ProjectTemplates::0.8.1908.202-beta" && \
+RUN dotnet new -i "Microsoft.Quantum.ProjectTemplates::0.9.1908.2906" && \
```

Comment:
It would be lovely if these version numbers were (possibly environment?) variables all set in one place.

Reply:
This version number is at least updated as a part of the all-QDK build, but we currently don't have a great mechanism in place for reflecting that version update within each individual repo except for with manual PRs.